### ISSUE TITLE
Symmetry in d3q27 is inconsistent

### ISSUE THREAD

**Issue by mdzik** [CFD-GO/TCLB#127]
I found that Symmetry in d3q27 is not working

Please fill:

My configuration:
[run.txt](https://github.com/CFD-GO/TCLB/files/668645/run.txt)

Same config work fine with d3q27_cummulant, faile with d3q27
Relevant symmetry code:
```C
CudaDeviceFunction void NSymmetry()
{
<?R Symmetry(2, -1) ?>
}

CudaDeviceFunction void SSymmetry()
{
<?R Symmetry(2, 1) ?>
}
```
is the same for both models :/

BUT to have it work in d3q27 one need to change N with S in XML. Ideas?

**Comment by mdzik** [comment]
@shkodm Do you have time to look at that?

**Comment by shkodm** [comment]
@mdzik, it's fine, I will try to look at it this week.

**Comment by shkodm** [comment]
Fixed via #129
